velox/exec/HashProbe.cpp

```diff
@@ -1079,6 +1087,30 @@ RowVectorPtr HashProbe::getOutputInternal(bool toSpillOutput) {
       return nullptr;
     }
 
+    if (accumulatedNumOutput > 0 ||
+        (filter_ != nullptr && numOut < numOutputBeforeFilter / 10 &&
+         !emptyBuildSide)) {
+      accumulatedNumOutput += numOut;
+      if (!resultIter_->atEnd() &&
+          accumulatedNumOutput < outputTableRowsCapacity_ &&
```

Comment:
`s/outputTableRowsCapacity_/outputBatchSize/` since `outputTableRowsCapacity_` counts extra space reserved for carryover row

Reply:
I double checked that `accumulatedNumOutput < outputTableRowsCapacity_` and the condiction `outputBatchSize - numOut > 0` are duplicate. I only kept the second condition.
Thanks.

Reply:
Can you explain why those 2 are duplicate? With
* numOut <= accumulatedNumOutput
* outputBatchSize <= outputTableRowsCapacity_
* numOut < outputBatchSize

how do you deduct
* accumulatedNumOutput < outputTableRowsCapacity_

?

Reply:
Before entering the for loop on line 1023, `outputTableRowsCapacity_` is set to `outputBatchSize + 1`. In each iteration, `numOut` is added to `accumulatedNumOutput`, and `numOut` is subtracted from `outputBatchSize`. Both conditions are  for checking if `outputTableRows` has sufficient capacity to ontinue. I think these two Therefore, I believe only the condition `outputBatchSize - numOut > 0` should be kept.